packages/unit-testing/README.md

```diff
@@ -32,14 +34,243 @@ module.exports = SuiteCloudJestConfiguration.build({
 ```
 
 ## Installation
+> ⚠ SuiteCloud Unit Testing is installed as a `devDependency`.
+
 To install SuiteCloud Unit Testing in your project, from your project folder, run the following command:
 ```
 npm install @oracle/suitecloud-unit-testing --save-dev
 ```
 
 ➡ If you use SuiteCloud CLI for Node.js, you can install SuiteCloud Unit Testing when running the [`project:create`](../../packages/node-cli/README.md/##Commands) command by following the questions prompted.
 
-⚠ SuiteCloud Unit Testing is installed as a `devDependency`.
+## SuiteCloud Unit Testing Examples
+
+Here you can find two examples on how to use SuiteCloud Unit Testing with a SuiteCloud project.
+
+The first example covers testing for the **N/record** module, which is fully mocked in SuiteCloud Unit Testing. Whereas the second example covers the testing of a module that is not mocked in SuiteCloud Unit Testing, by using a custom stub.
+
+>💡 You can manually mock any module that is still not supported in SuiteCloud Unit Testing.
+
+### N/record Module Example
+
+Make sure your project follows a structure similar to the one presented below:
+```
+myAccountCustomizationProject
+> __tests__
+	> Suitelet.test.js
+> src
+	> AccountConfiguration
+	> FileCabinet
+		> SuiteScripts
+      > Suitelet.js
+		> Templates
+		> Web Site Hosting Files
+	> Objects
+	> Translations
+  deploy.xml
+  manifest.xml
+suitecloud.config.js
+jest.config.js
+```
+
+To see the content of the files you need to make SuiteCloud Unit Testing work, check the following list:
```

Comment:
I think this sentence is a bit complicated I couldn't understand it right away, also it's not true that the user needs all these file in order for unit testing to work, only config files. Maybe smth. more simple like:
'Here's the content of the files in this example'